src/main/java/edu/unc/lib/boxc/migration/cdm/services/CdmIndexService.java

```diff
@@ -335,6 +360,52 @@ private String indexFieldType(String exportField) {
         }
     }
 
+    /**
+     * Indexes all exported objects for this project
+     * @param options
+     * @throws IOException
+     */
+    public void indexAllFromCsv(CdmIndexOptions options) throws IOException {
+        assertObjectsExported(options);
+
+        CdmFieldInfo fieldInfo = fieldService.loadFromCsvFieldsFromProject(project);
+        List<String> exportFields = fieldInfo.listAllExportFields();
+        outputLogger.info(exportFields.toString());
+        recordInsertSqlTemplate = makeInsertTemplate(exportFields);
+
+        try (
+                var conn = openDbConnection();
+                Reader reader = Files.newBufferedReader(options.getCsvFile());
+                CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT
+                        .withFirstRecordAsHeader()
+                        .withHeader(String.valueOf(exportFields))
+                        .withTrim());
+        ) {
+            for (CSVRecord csvRecord : csvParser) {
+                if (!csvRecord.get(0).isEmpty()) {
+                    List<String> fieldValues = new ArrayList<>();
+                    for (int i = 0; i < csvRecord.size(); i++) {
+                        fieldValues.add(csvRecord.get(i));
+                    }
```

Comment:
It looks like there is `csvRecord.toList()` which will do the same thing as this.